x-pack/plugins/ingest_manager/server/services/epm/registry/registry_url.ts

```diff
@@ -20,7 +20,14 @@ const SNAPSHOT_REGISTRY_URL_CDN = 'https://epr-snapshot.elastic.co';
 // const SNAPSHOT_REGISTRY_URL_NO_CDN = 'https://epr-snapshot.ea-web.elastic.dev';
 
 const getDefaultRegistryUrl = (): string => {
-  const branch = appContextService.getKibanaBranch();
+  let branch;
+  try {
+    branch = appContextService.getKibanaBranch();
+  } catch (error) {
```

Comment:
Under what conditions can kibana branch not be set?

Reply:
During the test:ftr Endpoint tests. The error seems in the ftr:runner portion.

On the phone right now, but check the stack trace in https://github.com/elastic/kibana/pull/73555#issuecomment-665784435

I can add details about what I saw while debugging later

It was a good reminder that those methods throw :(